### PR TITLE
Your first website

### PR DESCRIPTION
**Pull request by ghost**
Approve this Pull Request to get your basic site up and running.

### CODE REVIEW

**Comment by anwannpaul** [comment]
Okay

**Comment by ghost** [comment]
You need to merge it. :heart:

File: index.markdown [hunk 1]
```diff
@@ -0,0 +1,4 @@
+---
+title: Paul Anwan's Blog
```
Comment (anwannpaul) [inline index.markdown:2]: It's ANWANN.

Paul Anwann (there are three "N's" in the name. Thank you

Comment (ghost) [reply index.markdown:2]: I'm sorry, I was doing this in quite a hurry just now. :unicorn: :heart:

**Review by anwannpaul — APPROVED** [review]
The name is PAUL ANWANN (doesn't need to be in all caps)
But I'd prefer Paul Scythe. It's up to you 😉

File: index.markdown (hunk 1 above)
Comment (ghost) [reply index.markdown:2]: I'll give you exact instructions on how to edit and create posts, once this is merged.

**Pull request merged**